.gitignore

```diff
@@ -115,3 +115,9 @@ coverage.xml
 
 ## v8 crash reports
 isolate-*-v8.log
+
+## Prettier cache
+.prettier-cache
```

Comment:
Otherwise, goes to node-modules

Reply:
What's the problem with putting these in `node_modules`? That seems vastly preferable to cluttering the root of the repository with random cache directories.

Reply:
In CI, you then have to restore caches in a specific order (node_modules first) or your cache will be incorrect. I'm fine doing that, it just gets confusing.

Reply:
Though this raises a good point: now there is 4 dotfiles in the root dir for caching (d2, eslint, prettier, turbo).

Reply:
AFAIK we never actually cache `node_modules` as a whole, right? `actions/setup-node` caches the dependency manager's cache directory, which is not located in `node_modules`. Since we're using Yarn, it'll run `yarn config get cacheFolder` to get the cache location:

https://github.com/actions/setup-node/blob/a7c2d9473e135474f9c8e6d48969d1d90f950ea1/src/cache-utils.ts#L57C35-L57C62

On my machine, that's `/Users/nathan/.yarn/berry/cache`; I imagine it's something similar on the Actions runner.

Is there something else that's restoring a cache to `node_modules` that I'm not aware of? Or am I misunderstanding the problem?

Reply:
No, I'm misunderstanding. I setup a similar pipeline for another project where the caching was of node_modules. I didn't understand that the yarn cache is different. Sorry about that, I'll change this.

Reply:
Fixed in [7fd3235](https://github.com/PrairieLearn/PrairieLearn/pull/11544/commits/7fd32359b1f06454a0db089721cba7bb39bd9a40)